packages/server/database/migrations/20190912093945-remove-legacy-tables.ts

```diff
@@ -1,10 +1,13 @@
 exports.up = async (r) => {
   try {
-    await Promise.all([
-      r.tableDrop('Meeting').run(),
-      r.tableDrop('AtlassianProject').run(),
-      r.tableDrop('Outcome').run()
-    ])
+    await Promise.all([r.tableDrop('Meeting').run(), r.tableDrop('Outcome').run()])
+    const hasAtlassianProjectTable = await r
+      .tableList()
+      .contains('AtlassianProject')
```

Comment:
This could be re-written in ReQL as...

```javascript
r.tableList()
 .contains('AtlassianProject')
 .branch(r.tableDrop('AtlassianProject')).run()
```

...and you could preserve doing all of the database calls in one `Promise.all`.

Reply:
Ooh `.branch` is nice!

Updated in b57761b. Tested from scratch.